manticore/core/cpu/x86.py

```diff
@@ -1,48 +1,54 @@
-from capstone import *
-from capstone.x86 import *
-from .abstractcpu import Abi, SyscallAbi, Cpu, RegisterFile, Operand, instruction
-from .abstractcpu import ConcretizeRegister, ConcretizeRegister, ConcretizeArgument
-from .abstractcpu import Interruption, Syscall
-from functools import wraps
 import collections
+import logging
```

Comment:
changed the wildcard import for capstone to explicitly show where it is used.
The rest are minor whitespace fixes (most from vim)